### ISSUE TITLE
not working after upgrade to OS X Yosemite (10.10) 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [TeXworks/texworks#672]
```
After upgrade to OS X Yosemite (10.10) ! pdflatex does not work correctly when 
I add a novel test.eps file:

\begin{figure}
\includegraphics[width=0.5\textwidth]{test}
\caption{\label{Fig2} Lorem ipsum...}
\end{figure}

I get the following error:

! Package pdftex.def Error: File `test' not found.

See the pdftex.def package documentation for explanation.
Type  H <return>  for immediate help.
```

Original issue reported on code.google.com by `f.ronci2...@gmail.com` on 26 Oct 2014 at 5:17

**Comment by emilyemorehouse** [comment]
Any progress on this? I'm currently having this issue.

**Comment by UniqMartin** [comment]
Can you provide more information about your issue? What version of OS X are you running? What version of TeXworks and where did you get it from? How did you typeset to trigger the problem? What was the error message? Thanks!

As far as I can tell, Apple fixed the underlying issue in OS X 10.10.3, so everything should work as expected. I just ran a few tests on my system (OS X 10.10.3) and it worked, but maybe my test is flawed.

**Comment by emilyemorehouse** [comment]
I actually solved my problem, but for anyone who stumbles across this who may be having issues:

I'm on OS X 10.10.2 with a fresh install of MacTex-2014. 

I was actually trying to run a Sphinx make command to convert some Latex files to pdf, so I wasn't using any GUI. After ensuring that /usr/texbin had been added to my path, I found that the default installation of MacTex does some crazy symlinking from /usr/texbin. My problem was solved by updating a symlink:

``sudo ln -s /usr/texbin/pdflatex /usr/bin/pdflatex``

**Comment by UniqMartin** [comment]
While the symbolic link `/usr/texbin` might seem crazy as it goes through several indirections, there's a reason for that and it should work. I would not recommend putting symbolic links into `/usr/bin` as this is a system path and changes therein are not likely to survive OS updates.

In any case, your problem seems to be that `/usr/texbin` was in fact **not added** to your path, either by mistake or because of the bug in OS X 10.10 prior to 10.10.3. I think an update to 10.10.3 should rule out the latter issue.

**Comment by UniqMartin** [comment]
@stloeffler To the best of my knowledge, this problem (wrong PATH in environment due to a general problem in the environment handling) is solved in OS X 10.10.3. I think this issue can be closed.

**Comment by emilyemorehouse** [comment]
I've been putting off the upgrade to 10.10.3. I'll update back once I do.
